src/components/NcAvatar/NcAvatar.vue

```diff
@@ -405,6 +405,12 @@ export default {
 			}
 			return style
 		},
+		initialsStyle() {
+			const { r, g, b } = usernameToColor(this.getUserIdentifier)
+			return {
+				color: `rgba(${r}, ${g}, ${b})`
```

Comment:
```suggestion
				color: `rgb(${r}, ${g}, ${b})`
```
Since no 4th argument.